cli/schema/cypress.schema.json

```diff
@@ -278,6 +278,11 @@
           "type": "boolean",
           "default": false,
           "description": "Enables including elements within the shadow DOM when using querying commands (e.g. cy.get(), cy.find()). Can be set globally in cypress.json, per-suite or per-test in the test configuration object, or programmatically with Cypress.config()"
+        },
+        "babelLoaderOptions": {
```

Comment:
This is not needed, this file is for the options in `cypress.json`. [Docs here](https://docs.cypress.io/guides/references/configuration).

Reply:
@lmiller1990 ah ok, I had assumed that it could also be passed in via cypress.json (as the config would get forwarded) but it appears it is filtered out.

Is this still the right way to pass non global config? Or should it be in a separate param?

Reply:
I've reverted the change and pushed